docs/language-agnostic-definition.md

```diff
@@ -15,11 +15,19 @@ message Envelope{
     string uri = 5;
 }
 ```
-
+<!--
 <div class="admonition note">
   <p class="admonition-title">Note</p>
   <p>Additional constraints on envelope fields will be added soon!</p>
 </div>
+-->
+
+The format for the above fields, except `message`, is specified below. For those with `regexp`, the format is described in <a href="https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-language-quick-reference" target=_blank>regular expression.</a>.
+
+ * to: (`regexp`) `something`
```

Comment:
What is the regex for `to` and `sender`?

Also check the regex for `protocol_id` is right.

Reply:
We have not yet implemented the regex for `to` and `sender`. We need a regex which encapsulates valid addresses.

Reply:
ok